Python.gitignore

```diff
@@ -117,6 +117,10 @@ venv.bak/
 .spyderproject
 .spyproject
 
+
+# Pycharm
+/.idea
```

Comment:
As .idea is a directory, it should be treated as **.idea/** instead of **/.idea**, like you added in this commit. Correct me if I am wrong.

I purposed this change related to PyCharm configuration directory around 6 weeks ago in my PR. https://github.com/github/gitignore/pull/3769

Reply:
Leavea a commint

Reply:
Looks like your PR does what I wanted. *But will GitHub ever merge any of these PRs?*

I don't really modify gitignores, so I didn't know the formatting. Thanks for pointing that out!